File: 351003/Gorodko/app/Model/Editor.cs
namespace Project.Model {

    public class Editor : BaseEntity {
        public string Login { get; set; } = string.Empty;
        public string Password { get; set; } = string.Empty;
        public string Firstname { get; set; } = string.Empty;
        public string Lastname { get; set; } = string.Empty;

        public List<Tweet> Tweets { get; set; } = new();
    }

}

File: 351004/Nepesov/Lab6/untitled/src/main/java/org/example/model/Editor.java
package org.example.model;

import lombok.*;
import org.springframework.data.cassandra.core.mapping.Indexed;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;

@Table("tbl_editor")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Editor {

    @PrimaryKey
    private Long id;

    // Добавляем индекс, чтобы поиск existsByLogin работал без ошибок 500
    @Indexed
    private String login;

    private String password;

    private String firstname;

    private String lastname;
}